templates/package.twig

```diff
@@ -9,14 +9,15 @@
     </p>
     <h4 class="headline--alpha">About the event</h4>
     <p>
-        Speakers will be provided with a projector, a wireless lapel microphone and a screen
+        Speakers will be provided with a projector, a microphone and a screen
         for their presentation (size depends on the room). Speakers should bring any equipment
         they need to connect to projectors (VGA). It is also suggested that you reduce your
         dependency on the in-house internet connection as much as possible.
     </p>
     <p>
         When the Call for Papers closes, the organizers will sort through the submissions, making
-        their selections. Speakers will be notified as soon as the selection process has been completed.
+        their selections. The first stage of the speaker selection process is entirely anonomized.
```

Comment:
Nice! We didn't really publicly make a big deal about this, I think it'd be prudent to make more of a point of this fact this time.